media-sound/wavpack/wavpack-5.0.0.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="Open audio compression format"
+DESCRIPTION="WavPack is a completely open audio compression format, providing \
+lossless, high-quality lossy, and a unique hybrid compression mode."
+HOMEPAGE="http://wavpack.com/"
+COPYRIGHT="1998-2016 David Bryant"
+LICENSE="WavPack"
+REVISION="1"
+SOURCE_URI="http://wavpack.com/wavpack-$portVersion.tar.bz2"
+CHECKSUM_SHA256="918d7e32a19598df543b17fff840b10a0880f87296f9e32af454d256b6a64049"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	wavpack$secondaryArchSuffix = $portVersion
+	lib:libwavpack$secondaryArchSuffix = 1.2.0 compat >= 0
+	cmd:wavpack$secondaryArchSuffix = $portVersion
+	cmd:wvgain$secondaryArchSuffix = $portVersion
+	cmd:wvunpack$secondaryArchSuffix = $portVersion
```

Comment:
If you are sure it does not build on x86_gcc2 you can add the commandBinDir "trick" and replace
```
	cmd:wavpack$secondaryArchSuffix = $portVersion
	cmd:wvgain$secondaryArchSuffix = $portVersion
	cmd:wvunpack$secondaryArchSuffix = $portVersion
```
by
```
	cmd:wavpack = $portVersion
	cmd:wvgain = $portVersion
	cmd:wvunpack = $portVersion
```